.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET = "joddKartOP"
```

Comment:
suggestion - don't push .env files to github, add it to gitignore.

Reply:
forgot to put this in git ignore. Thanks for pointing out!